.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "saleor"]
+	path = saleor
+	url = git@github.com:mirumee/saleor.git
+	branch = master
```

Comment:
I don't think this is a good idea as it will break more often that not.

Reply:
you mean to fix it some version in particular? or abandon submodules as whole and just clone three separate repositories?

Reply:
This tool should use stable versions. The latest versions now are:
- core: 2.9.1
- storefront: 0.7.0
- dashboard: 2.0.0

See the "Releases" pages on Github in each repo for particular tags.